### PR TITLE
Feat: added routing+folder structure

### PR DESCRIPTION
**Pull request by coldpigli**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (sadathshariff) [inline public/index.html:27]: Might have forgotten to change the Title

**Review by sadathshariff — APPROVED** [review]
The Initial Setup looks good

File: src/routes/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+
+export{
+    Feed,
+    FindPeers,
+    Landing,
+    Login,
+    Signup,
+    UserProfile
+}
```
Comment (aishwarya-bm) [inline src/routes/index.js:16]: Instead of an explicit export statement here, you can replace import with export in lines 1-6

**Review by aishwarya-bm — APPROVED** [review]
Everything looks good for initial setup.

**Pull request merged**